operations/operations/publishing/publishing-guidelines/documentation-style-guide.md

```diff
@@ -0,0 +1,259 @@
+navigate the document. The table of contents should be added below the abstract. 
+
+### Introductory paragraph
+
+Each page should have an introduction that acts as a short description of the document. The short description
+should be a single paragraph of no more than three sentences. 
+
+## Using ReStructuredText markup (rst)
+
+The reStructuredText specification allows for a certain degree of flexibility in 
+markup to achieve your goals. For example, you can use any one of more than a dozen characters for 
+section title underlines, and you have the option of using an overline in addition to an underline.
+
+The majority of Mattermost technical documentation is written in `.rst`. However, there are some instances
+where Markdown is used. 
+
+Use the following conventions when writing in `.rst`. 
+
+### Page titles
+
+Underline page titles using `=`, with no overline. Underlines should be as long as the title text. For example:
+
+```
+  Document title
+  ==============
+```
+
+### Section titles
+
+Underline using `-` for section titles. For example:
+
+```
+  Section title
+  -------------
+``` 
+
+Underline subsections using `~` for the first subsection level, and `^` for the second subsection level. For example:
+
+```
+  Subsection one
+  ~~~~~~~~~~~~~~
+
+  Subsection two
+  ^^^^^^^^^^^^^^
+``` 
+
+### Table of contents
+
+Insert a table of contents into a document using the following format:
+
+```
+  .. contents::
+      :backlinks: top
+```  
+    
+### Inline Markup
+
+Bold
+  Use double asterisks. For example: `**bold text**`.
+
+Italic
+  Use single asterisk. For example: `*italic text*`.
+
+Monospace
+  Use double backquotes. For example: ```monospace text```
+
+### Bullet lists
+
+For bullet lists and sublists, use `-` before the list item. For example:
+
+```
+  - list item one
+  - list item two
+    - sublist item one
+    - sublist item two
+  - list item three
+```
+
+### Numbered lists and procedure steps
+
+Create numbered lists and procedure steps using numbers for the top-level list and lower case alpha characters for the first nested list. For example:
+
+```
+  1. This is item one.
+  2. This is item two.
+
+    a. This is nested item one.
+    b. This is nested item two.
+
+  3. This is item three.
+```
+
+### Name-value groups
+
+To create a name-value group such as a definition list, type the term on a line by itself. On the next line, indent the definition. For example:
+
+```
+  Total users
+    The total number of active accounts created on your system. Excludes inactive accounts.
+  Total teams
+    The total number of teams created on your system.
+```
+
+### External links
+
+URLs are automatically rendered as links in Sphinx; however, where possible, it is preferred that hyperlinks are created within the text of a sentence. Hyperlinks within a sentence can be created using the following formatting:
+
+`Link display text <URL-of-website>`__`, for example:
```

Comment:
```suggestion
`Link display text <URL-of-website>`__, for example:
```

Not sure if the last backstick is needed or if the intention is to have another backstick at the beginning.